### DISCUSSION TITLE
Upgrading Storage

### DISCUSSION THREAD

**Discussion by giripriyadarshan** [immich-app/immich#6882] [Q&A] [answered]
Hi, how to upgrade or add storage if the current upload path runs out of space?

**Comment by bo0tzz** [comment, marked as answer]
Move the folder to a larger drive

**Reply by giripriyadarshan** [reply]
any option to scale horizontally?

**Reply by bo0tzz** [reply]
What do you mean exactly? Managing storage space/drives isn't really an application-level task.

**Reply by giripriyadarshan** [reply]
How? ..... there is only 1 upload folder and optional unlimited external folders ...... so If I add a drive, can I keep the old upload folder as it is and mark the new drive as the upload path for any new uploads?

**Reply by schuhbacca** [reply]
No. You will have to have everything in one folder for the default upload location. That can be wherever you want though.

**Reply by giripriyadarshan** [reply]
Can this be marked as feature request please

**Reply by schuhbacca** [reply]
You can, but personally to me this seems outside of the scope. You can already store multiple external libraries in separate locations. As well as you can configure your storage however you want, as long as you can mount a singular folder into the container.

**Reply by giripriyadarshan** [reply]
The upload folder stores media in its own structure ... And separating according to users will make it a hassle...... It would be better if there was an option to change the upload folder when the previous drive's folder went out of storage ..... I want the sync option to work ... So imo this feature would help a lot to scale horizontally
